SR2019/2019-05-28-single-item-kit-chase.md

```diff
@@ -0,0 +1,27 @@
+---
+to: Teams with outstanding single items of kit
+subject: Outstanding kit
+---
+
+Hello,
+
+According to our records, you still have some items from the robot kit you borrowed during the SR2019 competition. As the competition has ended, we would like this back!
+
+At the end of the competition, we checked your kit in, and some items were missing. Below are the items we require back:
+
+## Missing items
+
+-
+-
+
+Please search anywhere you may have worked on your robot for these items. If you are unable to find any of these items, please [let us know](mailto:teams@studentrobotics.org).
+
+## Shipping items
+
+Please insure the kit during shipping and use a courier which supports tracking, to ensure the kit isn't lost.
+
+Please securely pack the kit and ship it to:
+
+1 Alie Street, Whitechapel, London, E1 8DE
+
+Once you have dispatched it, please [let us know](mailto:teams@studentrobotics.org). Please ensure your TLA is written on the outside of the packaging.
```

Comment:
D'oh, sorry. I've just noticed that you'd already asked for the school name to be on the outside of the parcel. I don't feel strongly about TLA vs school name.

I do however think it would make sense for the packaging instruction to appear before the after-post instruction. Much as we might hope otherwise, people are likely to do things in the same order as they read them.